scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/LiveJobs.java

```diff
@@ -812,6 +812,21 @@ private void setJobStatusAfterInErrorTaskRestart(InternalJob job) {
         }
     }
 
+    public static boolean isJobWithErrors(InternalJob job) {
+        switch (job.getStatus()) {
+            case FAILED:
+            case CANCELED:
+                return true;
+            default:
+                if (job.getJobInfo().getNumberOfFailedTasks() > 0 || job.getJobInfo().getNumberOfFaultyTasks() > 0) {
```

Comment:
The whole if-else part could be simplified to `return job.getJobInfo().getNumberOfFailedTasks() > 0 || job.getJobInfo().getNumberOfFaultyTasks() > 0`

Reply:
indeed